voronoi.cpp

```diff
@@ -143,19 +143,18 @@ void Voronoi::relax_points(int iterations = 1) {
 	std::vector<jcv_point> new_points;
 	new_points.reserve(_points.size());
 	for (int j = 0; j < iterations; j++) {
-		Ref<VoronoiDiagram> diagram { memnew(VoronoiDiagram) };
-		jcv_diagram_generate_useralloc(
+		jcv_diagram diagram;
+		memset(&diagram, 0, sizeof(jcv_diagram));
+		jcv_diagram_generate(
```

Comment:
I think using jcv_diagram_generate_useralloc was the correct move here, as it internally still allocates some memory.